platform-core-module/src/main/java/com/softicar/platform/core/module/maintenance/MaintenanceWindowPage.java

```diff
@@ -37,4 +43,21 @@ public EmfPagePath getPagePath(EmfPagePath modulePath) {
 
 		return modulePath.append(CoreI18n.MAINTENANCE);
 	}
+
+	@Override
+	public Collection<EmfPageBadge> getBadges(AGCoreModuleInstance moduleInstance) {
+
+		Supplier<Integer> inProgressCountSupplier = () -> AGMaintenanceWindow.TABLE//
+			.createSelect()
+			.where(AGMaintenanceWindow.STATE.isEqual(AGMaintenanceStateEnum.IN_PROGRESS.getRecord()))
+			.count();
+		Supplier<Integer> pendingCountSupplier = () -> AGMaintenanceWindow.TABLE//
+			.createSelect()
+			.where(AGMaintenanceWindow.STATE.isEqual(AGMaintenanceStateEnum.PENDING.getRecord()))
+			.count();
+		return List
+			.of(//
+				new EmfPageBadge(EmfDiagnosticLevel.WARNING, inProgressCountSupplier).setRefreshClasses(AGMaintenanceWindow.class),
+				new EmfPageBadge(EmfDiagnosticLevel.INFO, pendingCountSupplier).setRefreshClasses(AGMaintenanceWindow.class));
```

Comment:
How does this look?
one yellow badge with a number and a blue one with a number?
If so, see comment further down about labeling

Reply:
yes, see the demo video